rabblebot.rb

```diff
@@ -4,17 +4,25 @@
 
 # This is a simple discord bot that supports plugins
 class RabbleBot
-  attr_accessor :token, :app_id, :bot, :brain, :plugins
+  attr_accessor :token, :app_id, :bot, :brain, :plugins, :debug
 
   # expects you to set environment variables DISCORD_TOKEN and DISCORD_APP_ID
   # I used env vars because something something elastic beanstalk. (lol)
+  # if running locally set debug to true
   def initialize
-    @token = ENV.fetch('DISCORD_TOKEN')
-    @app_id = ENV.fetch('DISCORD_APP_ID')
+    @debug = false
+    if @debug
+      @token = 'DISCORD_TOKEN'
```

Comment:
You might want to consider writing a bash script to set the `DISCORD_TOKEN` and `DISCORD_APP_ID` for testing, instead of hard coding it